pkg/disttask/framework/storage/task_table.go

```diff
@@ -862,6 +897,9 @@ func (mgr *TaskManager) GetAllTasks(ctx context.Context) ([]*proto.TaskBase, err
 
 // GetAllSubtasks gets all subtasks with basic columns.
 func (mgr *TaskManager) GetAllSubtasks(ctx context.Context) ([]*proto.SubtaskBase, error) {
+	if err := injectfailpoint.DXFRandomError(0.01, errors.New("injected random error GetAllSubtasks")); err != nil {
```

Comment:
it seems the arguments are similar, we don't need to pass the probability as `failpoint` already support it, see
https://github.com/pingcap/failpoint?tab=readme-ov-file
```
[<percent>%][<count>*]<type>[(args...)][-><more terms>]
```
, and the error part we can pass a `tag` of string type only, we can let the inner method to compose the error

Reply:
> and the error part we can pass a tag of string type only, we can let the inner method to compose the error

we can even use `runtime.Caller` to get the caller function name automatically

Reply:
1. Even if failpoint supports it, it needs to pass the probability as an argument in the env variable. We may not use the same probability in all DXF failpoint.
2. Good idea